docs/editors/vim.md

```diff
@@ -245,70 +220,113 @@ Or use the default recommended mapping `<space> a`.
 This is helpful to see compilation errors in different files from your current
 open buffer.
 
-## Close buffer without exiting
+![Diagnostics](https://i.imgur.com/cer22HW.png)
 
-To close a buffer and return to the previous buffer, run the following command.
+## Run doctor
 
-```vim
-:bd
-```
+To troubleshoot problems with your build workspace, open your coc commands by either
+using `:CocCommand` or the recommend mapping `<space> c`. This will open your command
+window allowing you to search for `metals.doctor-run` command.
 
-This command is helpful when navigating in library dependency sources in the
-`.metals/readonly` directory.
+![Run Doctor Command](https://i.imgur.com/QaqhxF7.png)
 
-## Shut down the language server
+This command opens your browser with a table like this.
 
-The Metals server is shutdown when you exit vim as usual.
+![Run Doctor](https://i.imgur.com/yelm0jd.png)
 
-```vim
-:wq
-```
+## Other Available Command
 
-This step clean ups resources that are used by the server.
+  - `metals.restartServer`
+  - `metals.build-import`
+  - `metals.build-connect`
+  - `metals.sources-scan`
+  - `metals.compile-cascade`
+  - `metals.compile-cancel`
+  - `metals.doctor-run`
 
-## Run doctor
+## Show document symbols
 
-To troubleshoot problems with your build workspace, make sure the below snippet
-is in your `.vimrc` which will allow you to issue a `:MetalsDoctor` command.
+Run `:CocList outline` to show a symbol outline for the current file or use the
+default mapping `<space> o`.
 
-```vim
-command! -nargs=0 MetalsDoctor :call CocRequestAsync('metals', 'workspace/executeCommand', { 'command': 'doctor-run' })
-```
+![Document Symbols](https://i.imgur.com/gEhAXV4.png)
 
-This command opens your browser with a table like this.
 
-![Run Doctor](https://i.imgur.com/yelm0jd.png)
+## Available Configuration Options
+
+The following configuration options are currently available. The easiest way to set these configurations is to enter `:CocConfig` or `:CocLocalConfig` to set your global or local configuration settings respectively.
+If you'd like to get autocompletion help for the configuration values you can install [coc-json](https://github.com/neoclide/coc-json).
+
+
+   Configuration Option     |      Description
+----------------------------|---------------------------
+`metals.serverVersion`      | The version of the Metals server artifact. Requires reloading the window.
```

Comment:
This table can be automatically generated using mdoc similarly as it's done in `new-editor.md` https://github.com/scalameta/metals/blame/b1cc89c7b6bd081e772057b7bbb45377cc705796/docs/editors/new-editor.md#L312

It would be nice to avoid duplicating the descriptions here

Reply:
Ahh, good call. I'll update this to use mdoc.